ref_arch/multi_tenant.rst

```diff
@@ -0,0 +1,576 @@
+
+To add nodes, click "Change node count and size." A slider will appear for both the count and size. In this section we'll be changing only the count. You can learn more about other options in the [Cloud Scaling] section.
+
+.. image:: ../images/cloud-nodes-slider.png
+
+Drag the slider to increase node count by one, and click "Resize Formation." While the node is added the Cloud Console will display a message at the top of the screen:
+
+.. image:: ../images/cloud-change-progress.png
+
+.. note::
+
+  Don't forget that even when this process finishes there is more to do! The new node will be available in the system, but at this point no tenants are stored on it so **Citus will not yet run any queries there**.
+
+Node addition takes around five minutes. Refresh the browser until the change-in-progress message disappears. Next select the "Nodes" tab in the Cloud Console. You should see three nodes listed. Notice how the new node has no data on it (data size = 0 bytes):
+
+.. image:: ../images/cloud-node-stats.png
+
+To bring the node into play we can ask Citus to rebalance the data. This operation moves bundles of rows called shards between the currently active nodes to attempt to equalize the amount of data on each node. Rebalancing preserves :ref:`colocation`, which means we can tell Citus to rebalance the :code:`companies` table and it will take the hint and rebalance the other tables which are distributed by :code:`company_id`.
+
+Applications do not need to undergo downtime during shard rebalancing. Read requests continue seamlessly, and writes are locked only when they affect shards which are currently in flight.
+
+::
+
+  -- Spread data evenly between the nodes
+
+  SELECT rebalance_table_shards('companies');
+
+As it executes, the command outputs notices of each shard it moves:
+
+.. code-block:: text
+
+  NOTICE:  00000: Moving shard [id] from [host:port] to [host:port] ...
+
+Refreshing the Nodes tab in the Cloud Console shows that the new node now contains data! It can now help processing requests for some of the tenants.
+
+Dealing with Big Tenants
+------------------------
+
+The previous section describes a general-purpose way to scale a cluster as the number of tenants increases. However there's another technique that becomes important when individual tenants get especially large compared to the others.
+
+As the number of tenants increases, the size of tenant data typically tends to follow a Zipfian distribution. This means there are a few very large tenants, and many smaller ones. Hosting a large tenant together with small ones on a single worker node can degrade the performance for all of them.
+
+Performing standard Citus shard rebalancing may improve the mixing of large and small tenants, but this would be accidental because it is not designed to do so. The rebalancer simply distributes shards to equalize storage usage on nodes, without examining the ratio of shards allocated to one tenant vs another within any given node.
```

Comment:
Slight rephrasing for the first sentence "Performing standard Citus shard rebalancing will improve overall performance but it may or may not improve the mixing of large and small tenants".

Also, the second sentence doesn't seem right to me. It says "without examining the ratio of shards allocated to one tenant", where as in MT setup a single tenant will be allocated only one shard. So, maybe we say something like "The rebalancer simply .... , without examining which tenants are allocated on each shard".